tests/Fixture/TestContainer.ts

```diff
@@ -0,0 +1,29 @@
+import { tsParticles } from "../../src/index"; // Have to import this to setup custom window functions
```

Comment:
To prevent the unused import warning you can call tsParticles.init() in the constructor. This method does nothing if it was already called